libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -282,7 +272,7 @@ string YulUtilFunctions::shiftRightFunction(size_t _numBits)
 	// Note that if this is extended with signed shifts,
 	// the opcodes SAR and SDIV behave differently with regards to rounding!
 
-	string functionName = "shift_right_" + to_string(_numBits) + "_unsigned_" + m_evmVersion.name();
```

Comment:
I think it is still better to have the version here. This allows mixing code compiled for different evm versions without problems.

Reply:
@ekpyron and I talked about that case and didn't expect to see that happening. Is this something likely to happen?

Reply:
Thinking about it a bit more, we should probably either add it to all functions (including those that call the shift functions) or none. So let's not have it for now.